contracts/WETH10.sol

```diff
@@ -85,6 +88,15 @@ contract WETH10 {
 
         return true;
     }
+
+    function transferAndCall(address _to, uint _value, bytes memory _data) public returns (bool success) {
+        transferFrom(msg.sender, _to, _value);
+
+        if (isContract(_to)) {
```

Comment:
Does it make any sense to check if the receiver is a contract or an EOA?

Even if the receiver is a contract, there is no guarantee that with implement ERC667Receiver, so we might as well skip this check.